Docker-Swarm-deployment/single-node/nginx/example-configs/conf/ugix-html/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Data Exchange</title>
+  <link href="./style.css" rel="stylesheet">

```

Comment:
I think should be ``   <link href="style.css" rel="stylesheet">`` , this we changed in K8s ingress nginx , not have current directory ``/.`` . That also we need to commit :), maybe in another PR